lib/ceremony-dropdown.tsx

```diff
@@ -0,0 +1,108 @@
+import { Ceremony } from "@/types/types";
+import { DropdownMenuRadioItem } from "@/components/ui/dropdown-menu";
+import { Badge } from "@/components/ui/badge";
+import { useState, useEffect } from "react";
+import { getCeremonyList, getCeremonyPassports } from "./get-passport-data";
+
+
+export default function CeremonyDropdown() {
+    const [ceremonyList, setCeremonyList] = useState<Ceremony[]>([]);
+    const [cLoading, setCLoading] = useState(true);
+
+    useEffect(() => {
+        const fetchCeremonies = async () => {
+            setCLoading(true);
+            try {
+                const ceremonies = await getCeremonyList();
+                setCeremonyList(ceremonies || []);
+            } catch (error) {
+                setCeremonyList([]);
+            } finally {
+                setCLoading(false);
+            }
+        };
+
+        fetchCeremonies();
+    }, []);
+
+    if (cLoading) {
+        return <div>Loading...</div>;
+    }
+
+    return (
+        <>
+            {!(ceremonyList[0] == null) ? (
+                <>
+                    {ceremonyList.map((ceremony, index) => (
+                        <DropdownMenuRadioItem
+                            key={index}
+                            value={`ceremony-${ceremony.ceremony_time}`}
```

Comment:
it feels weird that this value is snake_case, i'd rather it be camelCase for consistency with other object properties in this repo

Reply:
The reason here is cause I'm pulling a value directly from the database, which uses snake_case (see auth.ts for another example)